test/e2e/log_size_monitoring.go

```diff
@@ -212,9 +215,11 @@ func (g *LogSizeGatherer) Work() bool {
 	)
 	if err != nil {
 		Logf("Error while trying to SSH to %v, skipping probe. Error: %v", workItem.ip, err)
+		workItem.backoffMultiplier *= 2
```

Comment:
Maybe some upper bound on it (so that it cannot exceed e.g. 100 or sth like that)?